src/test/scala/nl.knaw.dans.easy.bag2deposit/ddm/ProvenanceV2Spec.scala

```diff
@@ -0,0 +1,96 @@
+import javax.xml.transform.Source
+import javax.xml.transform.stream.StreamSource
+import javax.xml.validation.SchemaFactory
+
+class ProvenanceV2Spec extends AnyFlatSpec with FileSystemSupport with XmlSupport with Matchers with FixedCurrentDateTimeSupport with DebugEnhancedLogging with SchemaSupport with AppConfigSupport {
+  // use the raw github location while upgraded schema is not yet published, your own fork if not yet merged.
+  // TODO download from maven into target folder
+  override val schema: String = "target/dans-schema-lib/bag/metadata/prov/v2/provenance.xsd"
+
+  // FixedCurrentDateTimeSupport is not effective for a val
+  private def provenanceBuilder = Provenance("EasyConvertBagToDepositApp", "1.0.5")
+
+  "Provenance" should "change namespace" in {
+    val ddmIn = {
+      <ddm:DDM xmlns:ddm="http://easy.dans.knaw.nl/schemas/md/ddm/"
+               xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+               xsi:schemaLocation=" http://easy.dans.knaw.nl/schemas/md/ddm/ https://easy.dans.knaw.nl/schemas/md/ddm/ddm.xsd"
+               xmlns:gml="http://www.opengis.net/gml"
+      >
+        <ddm:profile>
+          <dc:title>blabla</dc:title>
+        </ddm:profile>
+      </ddm:DDM>
+    }
+
+    val expectedDdm = <ddm:DDM xmlns:ddm={ ddmV2namespace }
+             xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+             xmlns:gml="http://www.opengis.net/gml"
+    >
+      <ddm:profile>
+        <dc:title>blabla</dc:title>
+        <ddm:personalData present="Yes"/>
+      </ddm:profile>
+    </ddm:DDM>
+
+    val expectedProv = {
+      <prov:provenance xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+                       xmlns:prov="http://schemas.dans.knaw.nl/bag/metadata/prov/v2/"
+      >
+        <prov:migration app="EasyConvertBagToDepositApp" version="1.0.5" date="2020-02-02">
+        <prov:file scheme={ ddmV2namespace } xmlns=""
+                   xmlns:ddm={ ddmV2namespace }
+                   xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+                   xmlns:gml="http://www.opengis.net/gml"
+        >
+        <prov:old/>
+        <prov:new><ddm:personalData present="Yes"/></prov:new>
+          </prov:file>
+        </prov:migration>
+      </prov:provenance>
+    }
+
+    // a few steps of EasyConvertBagToDepositApp.addProps
+    val ddmOut = testConfig("archaeology").copy(ddmVersion = V2).ddmTransformer
+      .transform(ddmIn, "easy-dataset:123", containsPrivacySensitiveData = "true")
+      .getOrElse(fail("no DDM returned"))
+    val actualProv = provenanceBuilder.collectChangesInXmls(List(Provenance.compareDDM(ddmIn, ddmOut)))
+
+    // check what was stripped by normalize to avoid random order
+    ddmOut.serialize should include(ddmV2namespace)
+    actualProv.serialize should include(ddmV2namespace)
+
+    SchemaFactory
+      .newInstance(XMLConstants.W3C_XML_SCHEMA_NS_URI)
+      .newSchema(Array[Source](new StreamSource(new FileInputStream(schema))))
```

Comment:
This line results in 

    src-resolve: Cannot resolve the name 'dcterms:elementOrRefinementContainer' to a(n) 'type definition' component.
    org.xml.sax.SAXParseException; lineNumber: 124; columnNumber: 71; src-resolve: Cannot resolve the name 'dcterms:elementOrRefinementContainer' to a(n) 'type definition' component.

Reply:
While loading https://github.com/DANS-KNAW-jp/easy-convert-bag-to-deposit/blob/f4e835bfb0adf19b833b3c45c8b86c2693f37dac/src/test/scala/nl.knaw.dans.easy.bag2deposit/ddm/ProvenanceV2Spec.scala#L33
from https://github.com/DANS-KNAW-jp/easy-convert-bag-to-deposit/blob/f4e835bfb0adf19b833b3c45c8b86c2693f37dac/pom.xml#L83-L88